bindings/netframework/ElectionGuard.NetFramework/Tests/ElectionGuard.Tests/Mocks/ManifestGenerator.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using ElectionGuardInterop;
+
+namespace ElectionGuard.Tests.Mocks
+{
+    public class ManifestGenerator
+    {
+        public static Manifest GetMinimalFakeJeffersonCountyManifest()
+        {
+            var gpUnits = new List<GeopoliticalUnit>
+        {
+            new GeopoliticalUnit("jefferson-county", "Jefferson County", ReportingUnitType.county)
+        };
+
+            var candidates = new List<Candidate>
+        {
+            new Candidate("benjamin-franklin", isWriteIn: false),
+            new Candidate("john-adams", isWriteIn: false),
+            new Candidate("write-in", isWriteIn: true)
+        };
+
+            var contests = new List<ContestDescription>
+        {
+            new ContestDescription(
+                objectId: "justice-supreme-court",
+                electoralDistrictId: gpUnits[0].ObjectId,
+                sequenceOrder: 0,
+                VoteVariationType.n_of_m,
+                numberElected: 1,
+                name: "Justice of the Supreme Court",
+                selections: new[]
+                {
+                    new SelectionDescription(
+                        "benjamin-franklin-selection", candidates[0].CandidateId, sequenceOrder: 0),
+                    new SelectionDescription(
+                        "john-adams-selection", candidates[1].CandidateId, sequenceOrder: 1),
+                    new SelectionDescription(
+                        "write-in-selection", candidates[2].CandidateId, sequenceOrder: 2)
+                })
+        };
+
+            var ballotStyles = new List<BallotStyle>
+        {
+            new BallotStyle(
+                "jefferson-county-ballot-style",
+                new[] { gpUnits[0].ObjectId })
+        };
+
+
+            return new Manifest(
```

Comment:
this is the absolute bare minimum that is required for a manifest.  you'll notice that we create a party array and leave it blank.  Theres no enforced reference between parties and candidates in this example, but the constructor forces the object to exist to ensure the consumer of this knows it must be explicitly handled.